requirements.txt

```diff
@@ -4,5 +4,5 @@ Fabric==0.9.0
 nose==1.0.0
 rdflib==3.2.1
 argparse==1.1
--e git://github.com/mattbasta/fastchardet#egg=fastchardet
+fastchardet
```

Comment:
Please pin the version number: `fastchardet==0.2.0` (which is the latest version on pypi, and seems to also be the last commit on `master`, the one we've been using with the `-e` option)